app/novu/zod/schemas.ts

```diff
@@ -0,0 +1,56 @@
+import { z } from 'zod';
+
+// read more about zod at the official webside: https://zod.dev/
+export const emailListElementTypeSchema = z.object({title: z.string().default(""), body: z.string().default("")})
+
+export const componentTypeSchema = z.discriminatedUnion('componentType', [
+  z.object({ componentType: z.literal('text'), componentText: z.string().default(""), align:z.enum(["left", "center", "right"]).default("center")}),
+  z.object({ componentType: z.literal('list'), componentText: z.string().default(""), componentListItems: z.array(emailListElementTypeSchema)}),
+  z.object({ componentType: z.literal('image'), componentText: z.string().default(""), src: z.string().default("")}),
+  z.object({ componentType: z.literal('heading'), componentText: z.string().default("")}),
+  z.object({ componentType: z.literal('button'), componentText: z.string().default("")}),
+  z.object({ componentType: z.literal('divider')}),
+  z.object({ componentType: z.literal('users')}),
+]);
+
+export const zodControlSchema = z.object({
+  components: z.array(componentTypeSchema).default([{
+    "componentType": "heading",
+    "componentText": "Welcome to Novu"
+  }, {
+    "componentType": "text",
+    "componentText": "Congratulations on receiving your first notification email from Novu! Join the hundreds of thousands of developers worldwide who use Novu to build notification platforms for their products.",
+    "align": "left"
+  }, {
+    "componentType": "users"
+  }, {
+    "componentType": "list",
+    "componentListItems": [
+      {
+        title: "Send Multi-channel notifications",
+        body: "You can send notifications to your users via multiple channels (Email, SMS, Push, and In-App) in a heartbeat."
+      },
+      {
+        title: "Send Multi-channel notifications",
+        body: "You can send notifications to your users via multiple channels (Email, SMS, Push, and In-App) in a heartbeat."
+      }
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** Maybe something here about Delay and Digest?‏

Reply:
Added